docs/source/usage/parameters.rst

```diff
@@ -236,6 +236,20 @@ Initial Beam Distributions
         * ``beam.muypy`` (``float``, dimensionless, default: ``0``) correlation Y-Py
         * ``beam.mutpt`` (``float``, dimensionless, default: ``0``) correlation T-Pt
 
+    * ``thermal`` for a 6D stationary thermal or bithermal distribution.
+      This distribution type is described, for example in:
+      R. D. Ryne et al, "A Test Suite of Space-Charge Problems for Code Benchmarking", in Proc. EPAC2004, Lucerne, Switzerland.
+      C. E. Mitchell et al, "ImpactX Modeling of Benchmark Tests for Space Charge Validation", in Proc. HB2023, Geneva, Switzerland.
+      With additional parameters:
+
+        * ``beam.k`` (``float``, in inverse meters) external focusing strength
+        * ``beam.kT`` (``float``, dimensionless) temperature of core population
+        * ``beam.kT_halo`` (``float``, dimensionless, default ``kT``) temperature of halo population
+        * ``beam.normalize`` (``float``, dimensionless) normalizing constant for core population
+        * ``beam.normalize_halo`` (``float``, dimensionless) normalizing constant for halo population
```

Comment:
What is this normalizing constant used for and what is a good default value of this? `1.0`?

Reply:
Yes, a good default value is 1.0.  The explanation of this constant is too complex to provide here.  I will think about how best to document this.